### DISCUSSION TITLE
With new Standard Schema v0.40.1 - is yup supported?

### DISCUSSION THREAD

**Discussion by wilson12365** [TanStack/form#1067] [Q&A] [answered]
With new Standard Schema v0.40.1 - is Yup not supported?

**Comment by Balastrong** [comment, marked as answer]
If yup will support Standard Schema one day it would be awesome (https://github.com/jquense/yup/issues/2255) but in the meantime we'll try to keep it up with the usual adapter, it is still in the [docs](https://tanstack.com/form/latest/docs/framework/react/guides/validation#other-schema-libraries).

In short: we're still supporting Yup and there's no plan of dropping it in the near future :)
